xbmc/video/dialogs/GUIDialogAudioSubtitleSettings.cpp

```diff
@@ -293,9 +293,13 @@ void CGUIDialogAudioSubtitleSettings::InitializeSettings()
     g_application.m_pPlayer->GetSubtitleCapabilities(m_subCaps);
   }
 
+  // register IsPlayingPassthrough condition
+  m_settingsManager->AddCondition("IsPlayingPassthrough", IsPlayingPassthrough);
+
   CSettingDependency dependencyAudioOutputPassthroughDisabled(SettingDependencyTypeEnable, m_settingsManager);
-  dependencyAudioOutputPassthroughDisabled.And()
-    ->Add(CSettingDependencyConditionPtr(new CSettingDependencyCondition(SETTING_AUDIO_PASSTHROUGH, "false", SettingDependencyOperatorEquals, false, m_settingsManager)));
+  dependencyAudioOutputPassthroughDisabled.Or()
```

Comment:
Just tested Gotham 13.2 and it has the same problem but there the passthrough state is only applied to volume and volume amplification is always enabled. But the enabled/disabled state of volume is only correct after opening the dialog. After that when e.g. switching to passthrough it stays on and when disabling passthrough again it becomes disabled.

I'll have to think about a way to work around this.